packages/components/src/components/RadioCard/RadioCard.tsx

```diff
@@ -42,17 +47,28 @@ const Wrapper = styled.div<
     background: ${({ theme }) => theme.baseFillSurfacePage};
     ${({ onClick }) => onClick && 'cursor: pointer;'}
 
-    &:hover,
-    &:focus {
-        outline-color: ${({ theme }) => theme.baseContentEmphasis};
-    }
+    ${({ $isDisabled }) =>
+        !$isDisabled &&
+        css`
+            &:hover,
+            &:focus {
+                outline-color: ${({ theme }) => theme.baseContentEmphasis};
```

Comment:
pls, add this option to storybook. I checked it there and visually it looks okay. Altough, final approval should be on @jvaclavik 

And, in Storybook, make sure you define the type of this prop. Currently, you have to manually write it as an object. It should be a boolean switch.

Thanks